src/test/java/hw3/steps/HomePageSteps.java

```diff
@@ -0,0 +1,55 @@
+package hw3.steps;
+
+import hw3.pages.HomePage;
+import org.openqa.selenium.WebElement;
+
+import java.io.IOException;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertTrue;
+
+public abstract class HomePageSteps {
```

Comment:
all assertions must get as parameter WebElement/List not PO. Refactor all assertions.

Moreover. For what this class? Use BaseTest to common assertions or create special methods in PO.